setup.py

```diff
@@ -68,7 +68,7 @@
     "nncf": ["nncf>=2.16.0"],
     "openvino": ["nncf>=2.16.0", "openvino>=2025.1.0", "openvino-tokenizers>=2025.1.0"],
```

Comment:
adding before official support of openvino is available
```suggestion
    "openvino": ["nncf>=2.16.0", "openvino>=2025.1.0", "openvino-tokenizers>=2025.1.0", "transformers<4.52"],
```